### PR TITLE
multitenant: check query rows result in admin function tests

### PR DESCRIPTION
**Pull request by ecwall**
Informs CRDB-16746

Some admin functions return error rows in the response instead of an error itself. Update related tests to check these rows.

Release note: None

### CODE REVIEW

File: pkg/sql/multitenant_admin_function_test.go [hunk 1]
```diff
@@ -63,148 +63,233 @@ func createSecondaryTenantDB(
...
 			query:                    "ALTER RANGE RELOCATE VOTERS FROM 1 TO 1 FOR (SELECT min(range_id) FROM [SHOW RANGES FROM TABLE t]);",
+			expectedSystemResult:     [][]string{{"54", "/Table/53", "trying to add a voter to a store that already has a VOTER_FULL"}},
+			expectedSecondaryResult:  [][]string{{"55", "", rangeErrorMessage}},
 			skipSQLSystemTenantCheck: true,
 		},
 		{
-			desc:                 "ALTER TABLE x EXPERIMENTAL_RELOCATE LEASE",
-			query:                "ALTER TABLE t EXPERIMENTAL_RELOCATE LEASE SELECT 1, 1;",
-			expectedErrorMessage: errorutil.UnsupportedWithMultiTenancyMessage,
+			desc:                          "ALTER RANGE x RELOCATE NONVOTERS",
```
Comment (ecwall) [inline pkg/sql/multitenant_admin_function_test.go:171]: Also adds NONVOTERS version of VOTERS tests.

File: pkg/sql/multitenant_admin_function_test.go [hunk 2]
```diff
@@ -63,148 +63,233 @@ func createSecondaryTenantDB(
...
 		},
 		{
-			desc:                 "ALTER RANGE x RELOCATE VOTERS",
-			query:                "ALTER RANGE (SELECT min(range_id) FROM [SHOW RANGES FROM TABLE t]) RELOCATE VOTERS FROM 1 TO 1;",
-			expectedErrorMessage: errorutil.UnsupportedWithMultiTenancyMessage,
+			desc:                          "ALTER RANGE x RELOCATE VOTERS",
+			query:                         "ALTER RANGE (SELECT min(range_id) FROM [SHOW RANGES FROM TABLE t]) RELOCATE VOTERS FROM 1 TO 1;",
+			expectedSecondaryErrorMessage: errorutil.UnsupportedWithMultiTenancyMessage,
 		},
+		// TODO(ewall): Set up test to avoid "trying to add a voter to a store that already has a VOTER_FULL" error.
```
Comment (ecwall) [inline pkg/sql/multitenant_admin_function_test.go:149]: Some tests require additional setup that I will address when working on adding the Class to the related KV AdminRequest.

**Review by knz — APPROVED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/93225)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @arulajmani and @rafiss)

**Comment by ecwall** [comment]
bors r=knz

**Pull request merged**